src/runner/Bonds.jl

```diff
@@ -0,0 +1,87 @@
+###
+# BONDS
+###
+
+module Bonds
```

Comment:
This could be a breaking change (at least for my funky notebooks), and I don't think that that is worth it